delivery_postlogistics_server_env/README.rst

```diff
@@ -13,15 +13,15 @@ Delivery Postlogistics Server Environment
 .. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fdelivery--carrier-lightgray.png?logo=github
-    :target: https://github.com/OCA/delivery-carrier/tree/14.0/delivery_postlogistics_server_env
-    :alt: OCA/delivery-carrier
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fdelivery_postlogistics_server_env-lightgray.png?logo=github
+    :target: https://github.com/OCA/delivery_postlogistics_server_env/tree/16.0/delivery_postlogistics_server_env
+    :alt: OCA/delivery_postlogistics_server_env
 .. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
-    :target: https://translation.odoo-community.org/projects/delivery-carrier-14-0/delivery-carrier-14-0-delivery_postlogistics_server_env
+    :target: https://translation.odoo-community.org/projects/delivery_postlogistics_server_env-16-0/delivery_postlogistics_server_env-16-0-delivery_postlogistics_server_env
```

Comment:
`delivery_postlogistics_server_env` should be `delivery-carrier`

Reply:
Thank you.